proto/chart/v1/stream.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+option go_package = "chart/stream/v1;charts";
+
+package chart.v1;
+
+import "google/protobuf/timestamp.proto";
+
+message StreamChartsAndDimensions {
+  // claim_id, node_id pair is used to identify the Node Instance
+  string claim_id = 1;
+  string node_id = 2;
+  // used to define the position of the Cloud
```

Comment:
You sure this comment goes here? :thinking:

Reply:
Mm nah you're right the ones below should suffice.
In a way I wanted to use this as a kind of title for the rest of the fields below it. But, looking at it again I think it is a bit misleading.

Reply:
Deleted it 👍🏼